src/send-data.ts

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 import { CoverageJson, PRData } from './types';
 
-const sendDataComment = async (url: string, prData: PRData): Promise<string> => {
+const sendData = async (url: string, prData: PRData): Promise<PRData> => {
```

Comment:
Can we change the name of this type `PRData` Maybe use something a bit more descriptive. Like PullRequest. Also the function name `sendData` should be changed to something more explicit. Like which data is being sent.